### ISSUE TITLE
Got error 'No information available about transaction' when calling rpc getRawTransaction

### ISSUE THREAD

**Issue by rlinxy** [Qitmeer/qitmeer-wallet#54]
Occasionally got this error  'No information available about transaction' when calling the rpc API 'getRawTransaction', while the transaction was successfully sent out and can be tracked at the pmeer blockchain browser. It happens a couple of times per month.

2020/07/18 12:20:24 Failed to get tx confirmations: No information available about transaction f9ede9b8028420c9a7c3c6ffd0ff10672dd637d676bc66deb016485d9bb9dd60

**Comment by rlinxy** [comment]
To be clear, this wallet api works fine for most of the time, we got correct response after called 'getRawTransaction'. But for some 'special' tx such as the one from above, it throws that error.

**Comment by dindinw** [comment]
###  The block & tx can be found at 
- block:
https://explorer.qitmeer.io/block/98aee6410888e20409435cf8257e0a57e81ab4653cf72d36e24c039c8526ae91
- tx:
https://explorer.qitmeer.io/tx/f9ede9b8028420c9a7c3c6ffd0ff10672dd637d676bc66deb016485d9bb9dd60
### Test getRawTransaction by curl again 
I doubt it a wallet issue.  Please test it again sending `getRawTransaction` rpc by curl command to your backend `qitmeer` node.
```
curl -s -k -u user:password -X POST -H 'Content-Type: application/json' --data '{"jsonrpc":"2.0","method":"getRawTransaction","params":["f9ede9b8028420c9a7c3c6ffd0ff10672dd637d676bc66deb016485d9bb9dd60",true],"id":1}' https://127.0.0.1:18131
```
### Provide your qitmeer node version
Please also provide the qitmeer version details. which can be found using the `qitmeer --version` command.  or using `getNodeInfo` rpc
```
curl -s -k -u user:password -X POST -H 'Content-Type: application/json' --data '{"jsonrpc":"2.0","method":"getNodeInfo","params":[],"id":null}' https://127.0.0.1:18131|jq .buildversion
```

**Comment by dindinw** [comment]
@rlinxy  I think your issue might be related to https://github.com/Qitmeer/qitmeer/pull/327 。Please clean up your wallet's db and sync block again because your wallet db might contain some incorrect tx hash caused by an RPC bug in qitmeer 0.9.0. if you never clean up your wallet db since 0.9.0, you can do a fresh block synchronization to fix the issue.

**Comment by rlinxy** [comment]
My qitmeer version is:
qitmeer version 0.9.1+dev- (Go version go1.13.4)
The wallet db has been deleted and rebuilt after I updated to this version 0.9.1.

I guess it's not a new bug, because we have seen this error during the past several months before and after the recent fork.

**Comment by dindinw** [comment]
@rlinxy Please update your backend qitmeer node version to [0.9.1+hotfix-release-6e0a73a](https://github.com/Qitmeer/qitmeer/releases/tag/v0.9.1-release-hotfix) . and make sure to redo the block synchronization by clean up the qitmeer node db  ( using command `qitmeer --testnet --cleanup`). You might also need to rebuild your wallet db since the backend qitmeer node db updated.

**Comment by rlinxy** [comment]
OK, just cleaned up the data of both pmeer node and wallet, then updated the node to the newest version, let's see if this bug got fixed. Thanks.

**Comment by forchain** [comment]
@rlinxy bug fixed on latest build?

**Comment by forchain** [comment]
@rlinxy a bug from Qitmeer and probably gets fixed in Qitmeer/qitmeer#327, close first, please reactivate if happens some day
